src/components/Stack.tsx

```diff
@@ -9,6 +9,7 @@ type StackProps = {
   align?: "start" | "center" | "end" | "baseline";
   justifyContent?: "center" | "space-around" | "space-between";
   className?: string | boolean;
+  style?: object;
```

Comment:
```suggestion
  style?: React.CSSProperties;
```
Might be better to narrow type this.

Reply:
Thanks!

Reply:
Typescript is complaining though :/

Reply:
umm odd, will have a look.